go.mod

```diff
@@ -53,3 +51,7 @@ replace k8s.io/mount-utils => k8s.io/mount-utils v0.21.5-rc.0
 replace k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.21.4
 
 replace k8s.io/sample-controller => k8s.io/sample-controller v0.21.4
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.4
```

Comment:
This replace structure is odd. Perhaps go mod tidy fixes them?

Reply:
Fixed it manually. It's the script that downloads the k8s deps which made it look funny.